lib/features/manage_account/presentation/mailbox_visibility/mailbox_visibility_view.dart

```diff
@@ -0,0 +1,213 @@
+              minSize: 12,
+              icon: SvgPicture.asset(
+                  categories.getExpandMode(controller.mailboxCategoriesExpandMode.value) == ExpandMode.EXPAND
+                      ? _imagePaths.icExpandFolder
+                      : _imagePaths.icCollapseFolder,
+                  color: AppColor.primaryColor,
+                  fit: BoxFit.fill),
+              tooltip: AppLocalizations.of(context).collapse,
+              onTap: () => controller.toggleMailboxCategories(categories)),
+          Expanded(child: Text(categories.getTitle(context),
+              maxLines: 1,
+              overflow: TextOverflow.ellipsis,
+              style: const TextStyle(
+                  fontSize: 17,
+                  color: Colors.black,
+                  fontWeight: FontWeight.bold))),
+        ]));
+  }
+
+  Widget _buildMailboxCategory(BuildContext context, MailboxCategories categories, MailboxNode mailboxNode) {
+    if (categories == MailboxCategories.exchange) {
+      return Padding(
+        padding: EdgeInsets.symmetric(horizontal: _responsiveUtils.isDesktop(context) ? 24 : 0),
+        child: _buildBodyMailboxCategory(context, categories, mailboxNode),
+      );
+    }
+    return Padding(
+      padding: EdgeInsets.symmetric(horizontal: _responsiveUtils.isDesktop(context) ? 24 : 0),
+      child: Column(children: [
+        _buildHeaderMailboxCategory(context, categories),
+        AnimatedContainer(
+            duration: const Duration(milliseconds: 400),
+            child: categories.getExpandMode(controller.mailboxCategoriesExpandMode.value) == ExpandMode.EXPAND
+                ? _buildBodyMailboxCategory(context, categories, mailboxNode)
+                : const Offstage())
+      ]),
+    );
+  }
+
+  Widget _buildBodyMailboxCategory(
+    BuildContext context,
+    MailboxCategories categories,
+    MailboxNode mailboxNode,
+  ) {
+    final lastNode = mailboxNode.childrenItems?.last;
+
+    return Container(
+        padding: EdgeInsets.only(
+            right: _responsiveUtils.isDesktop(context) ? 0 : 12,
+            left: _responsiveUtils.isDesktop(context) ? 0 : 12),
+        child: TreeView(
+            key: Key('${categories.keyValue}_mailbox_list'),
+            children: _buildListChildTileWidget(
+                context,
+                mailboxNode,
+                lastNode: lastNode)));
+  }
+
+  List<Widget> _buildListChildTileWidget(
+    BuildContext context,
+    MailboxNode parentNode,
+    {MailboxNode? lastNode}
+  ) {
+    return parentNode.childrenItems
+        ?.map((mailboxNode) => mailboxNode.hasChildren()
+          ? TreeViewChild(
+              context,
+              key: const Key('children_tree_mailbox_child'),
+              isExpanded: mailboxNode.expandMode == ExpandMode.EXPAND,
+              parent: Obx(() => controller.defaultMailboxIsNotEmpty ? (MailBoxVisibilityFolderTileBuilder(context, _imagePaths, mailboxNode, lastNode: lastNode)
```

Comment:
why only check `defaultMailboxIsNotEmpty`? What about `personalMailbox` and `teamMailbox`?